examples/subscribe_notify_characteristic.rs

```diff
@@ -0,0 +1,122 @@
+}
+
+/**
+If you are getting run time error like that :
+ thread 'main' panicked at 'Can't scan BLE adapter for connected devices...: PermissionDenied',
+ you can try to run app with > sudo ./discover_adapters_peripherals
+ on linux
+**/
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn Error>> {
+
+    let manager = Manager::new().await?;
+    let adapter_list = manager.adapters().await?;
+    if adapter_list.len() <= 0 {
+        eprint!("Bluetooth adapter(s) were NOT found, sorry...\n");
+    } else {
+        for adapter in adapter_list.iter() {
+            println!("connecting to BLE adapter...");
+            print_adapter_info(&adapter);
+
+            adapter
+                .start_scan()
+                .await
+                .expect("Can't scan BLE adapter for connected devices...");
+            time::sleep(Duration::from_secs(2)).await;
+            let peripherals = adapter.peripherals().await?;
+
+            if peripherals.is_empty() {
+                eprintln!("->>> BLE peripheral devices were not found, sorry. Exiting...");
+            } else {
+                // all peripheral devices in range
+                for peripheral in peripherals.iter() {
+                    let properties = peripheral.properties().await?;
+                    let is_connected = peripheral.is_connected().await?;
+                    let local_name = properties.local_name.unwrap_or(String::from("Unknown prop name"));
+                    println!("peripheral : {:?} is connected: {:?}",
+                             &local_name, is_connected);
+                    // filter needed peripheral
+                    if local_name.contains(PERIPHERAL_NAME_MATCH_FILTER) {
+                        println!("found matching peripheral : {:?}...", &local_name);
+                        if !is_connected { // we can be connected previously
+                            let connect_result = peripheral
+                                .connect()
+                                .await;
+                            match connect_result {
+                                Ok(_) => {}
+                                Err(err) => {
+                                    eprintln!("Can't connect to peripheral, skipping due to error = {:?}...", err);
+                                    continue;
+                                }
+                            }
+                        }
+                        let is_connected = peripheral.is_connected().await?;
+                        println!(
+                            "now connected (\'{:?}\') to peripheral : {:?}...",
+                            is_connected, &local_name
+                        );
+                        let chars = peripheral.discover_characteristics().await?;
+                        if is_connected {
+                            println!(
+                                "Discover peripheral : \'{:?}\' characteristics...",
```

Comment:
Won't this result in the name being quoted twice? Get rid of the extra quote marks.